templates/blocks/spreadsheet.twig

```diff
@@ -0,0 +1,25 @@
+{% block spreadsheet %}
+
+	{% if ( fields.rows ) %}
+		<input class="spreadsheet-search" type="text" placeholder="{{ __('Search data') }}"/>
```

Comment:
can you also add `form-control` here as a class, to inherit some of our default styling?